org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/InvisibleProjectBuildSupport.java

```diff
@@ -65,7 +65,7 @@ public boolean fileChanged(IResource resource, CHANGE_TYPE changeType, IProgress
 	}
 
 	public boolean matchPattern(IPath base, String pattern, String path) {
-		return SelectorUtils.matchPath(ProjectUtils.resolveGlobPath(base, pattern).toString(), path);
+		return SelectorUtils.matchPath(ProjectUtils.resolveGlobPath(base, pattern).toOSString(), path);
```

Comment:
Resolved glob (`IPath`) now always use `toOSString` to convert to `String` type, instead of using `/` uniformly, because `SelecetorUtils.matchPath` will report instable result with '/' seperator in Windows.